src/Entities/Transaction.php

```diff
@@ -139,6 +147,54 @@ public function getPaymentCardId()
     }
 
     /**
+     * @return array
+     */
+    public function getChildTransactions()
+    {
+        return $this->getAttribute('childTransactions');
+    }
+
+    /**
+     * @return array
+     */
+    public function getInvoiceIds()
+    {
+        return $this->getAttribute('invoiceIds');
+    }
+
+    /**
+     * @return array
+     */
+    public function getSubscriptionIds()
+    {
+        return $this->getAttribute('subscriptionIds');
+    }
+
+    /**
+     * @return PaymentMethodInstrument
+     */
+    public function getPaymentInstrument()
+    {
+        return $this->getAttribute('paymentInstrument');
+    }
+
+    /**
+     * @return bool
+     */
+    public function hasDcc()
+    {
+        return $this->getAttribute('hasDcc');
+    }
+
+    /**
+     * @return array
+     */
+    public function getDcc()
+    {
+        return $this->getAttribute('dcc');
```

Comment:
This one seems strange considering there is an object in `dcc`... (I guess the same above with `paymentInstrument`)...